VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -44,6 +50,21 @@ public void setAppNameContext() {
             this.properties.put(Keys.CONTEXT, getAppNameContext());
         }
 
+        public void setCustomDimensions() {
```

Comment:
The whole purpose of the function is to set custom dimensions, so, why name it in any other way?
>Actually, since this method seems to only be called from the constructor, we might as well port the logic there directly I guess.

Having a function like such, will make it easy for us to add more dimensions in the future, as we'll have a whole function for it and won't have to search the code block/lines where to add them.